Added multimonitor support

SG420 opened the pull request:
Randomly sets a different wallpaper for each connected monitor. Makes use of swaymsg to list the connected monitors, then loops through the section in the main function that selects & sets the wallpaper

SG420 committed:
Added multimonitor support

Randomly sets a different wallpaper for each connected monitor

cycneuramus commented on swaywall:
```diff
@@ -75,19 +85,21 @@ def main():
-        set_wall(new)
+    outputs = get_outputs()
+    for output in outputs:
+        hst = get_history(hst_file)
+        if args.restore and hst:
+            set_wall(hst[0], output)
+            exit()
```
This will behave strangely since you have introduced multiple monitors but not, it seems, handled the wallpaper history.

cycneuramus commented on swaywall:
```diff
@@ -75,19 +85,21 @@ def main():
+            set_wall(hst[0], output)
+            exit()
+
+        walls = find(walls_dir, img_exts)
+        if not walls:
+            raise FileNotFoundError(f"no wallpapers found in {walls_dir}")
```
This should not be inside the for loop (as otherwise, of course, these lookups will be unnecessarily performed as many times as there are outputs).

cycneuramus reviewed (commented):
Hi, and thanks for contributing. Unfortunately, I cannot merge these changes as they stand since they would introduce several unwanted behaviors, the most conspicuous one being total breakage from not at all handling the wallpaper history for multiple monitors.

SG420 committed:
Fixed issues with multimonitor history

- Main function modified to remove unnecessary lookups
- Main function modified to handle multiple wallpapers history
- Remember function updated to handle multiple wallpapers being
  remembered

SG420 commented:
I have pushed some changes to fix these issues, tested with my dual-monitor setup and the restore option now works as expected. History file is also now adding both monitors' wallpapers as it should.
